File: 3783-mirror-distance-of-an-integer/3783-mirror-distance-of-an-integer.cpp
class Solution {
public:
    int mirrorDistance(int n) {
        int rev = 0, num = n;

        while(num != 0) {
            rev *= 10;
            rev += num % 10;
            num /= 10;
        }

        return n > rev ? n - rev : rev - n;
    }
};

File: 3783-mirror-distance-of-an-integer/3783-mirror-distance-of-an-integer.py
class Solution(object):
    def mirrorDistance(self, n):
        """
        :type n: int
        :rtype: int
        """
        rev, num = 0, n

        while num:
            rev *= 10
            rev += num % 10
            num //= 10

        return n - rev if n > rev else rev - n